tests/phpunit/tests/media.php

```diff
@@ -3054,16 +3053,17 @@ public function test_wp_filter_content_tags_loading_lazy() {
 			$iframe_eager,
 			$iframe_no_width_height
 		);
-		$content_filtered = wp_img_tag_add_decoding_attr( $content_filtered, 'the_content' );
 
 		// Do not add width, height, srcset, and sizes.
 		add_filter( 'wp_img_tag_add_width_and_height_attr', '__return_false' );
 		add_filter( 'wp_img_tag_add_srcset_and_sizes_attr', '__return_false' );
+		add_filter( 'wp_img_tag_add_decoding_attr', '__return_false' );
 
 		$this->assertSame( $content_filtered, wp_filter_content_tags( $content_unfiltered ) );
 
 		remove_filter( 'wp_img_tag_add_width_and_height_attr', '__return_false' );
 		remove_filter( 'wp_img_tag_add_srcset_and_sizes_attr', '__return_false' );
+		remove_filter( 'wp_img_tag_add_decoding_attr', '__return_false' );
```

Comment:
This test is currently failing. I believe that is because the filter is being set to `__return_false` too late. Further up in the test, `wp_img_tag_add_loading_optimization_attrs()` is already called on the expected result so that will add `decoding="async"`. So I think once you move the `add_filter( 'wp_img_tag_add_decoding_attr', '__return_false' );` line before that, it should pass as expected.